src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -285,18 +285,21 @@ export class ExposureNotificationService {
     };
 
     try {
-      await this.loadExposureStatus();
-      await this.loadExposureHistory();
-      await this.loadDisplayExposureHistory();
-      await this.updateExposureStatus();
       await this.processNotification();
       const qrEnabled = (await this.storageService.retrieve(StorageDirectory.GlobalQrEnabledKey)) === '1';
-      if (qrEnabled) {
-        const outbreakService = await OutbreakService.sharedInstance(this.i18n, this.backendInterface);
-        await outbreakService.checkForOutbreaks();
-      }
 
       const exposureStatus = this.exposureStatus.get();
+
+      if (!isDiagnosed(exposureStatus.type)) {
```

Comment:
This is handled already for proximity exposures i.e. 
```
if (updatedExposure.type === ExposureStatusType.Diagnosed) {
```

We should be able to do a check in "check for outbreaks" and leave the existing proximity checking code as is